engine/docs-runner/src/main/scala/org/enso/docsrunner/Main.scala

```diff
@@ -0,0 +1,50 @@
+package org.enso.docsrunner
+
+import java.io.File
+import org.enso.interpreter.runtime.{Context => EnsoContext}
+import org.enso.languageserver.io.{
+  ObservableOutputStream,
+  ObservablePipedInputStream
+}
+import org.enso.polyglot.{LanguageInfo, MethodNames}
+import org.graalvm.polyglot.Context
+
+object Main {
+  val stdOut    = new ObservableOutputStream
+  val stdErr    = new ObservableOutputStream
+  val stdInSink = new ObservableOutputStream
+  val stdIn     = new ObservablePipedInputStream(stdInSink)
+
+  def generateFrom(file: File): Unit = {
+    val executionContext = Context
+      .newBuilder()
+      .allowAllAccess(true)
+      .allowExperimentalOptions(true)
+      .out(stdOut)
+      .err(stdErr)
+      .in(stdIn)
+      .build()
+
+    val languageContext = executionContext
+      .getBindings(LanguageInfo.ID)
+      .invokeMember(MethodNames.TopScope.LEAK_CONTEXT)
```

Comment:
This is wrong – `LEAK_CONTEXT` should never be used in production code, it's an internal hack used for testing only.

Reply:
I thought the name `LEAK` is scary enough to prevent this :P